app/views/smart_proxies/logs/_modal.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="modal fade" id="backtrace-modal" tabindex="-1" role="dialog" aria-labelledby="bt-modal-label">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="bt-modal-label"><%= _('Log entry details') %></h4>
+      </div>
+      <div class="modal-body">
+        <table class='<%= table_css_classes %> table-fixed'>
+        <tbody>
+          <tr>
+              <td class="nbsp col-md-2"><%= _('Local time') %></td>
+              <td class="col-md-10"><div id="modal-bt-time"/></td>
+          </tr>
+          <tr>
+              <td class="nbsp"><%= _('GMT') %></td>
+              <td><div id="modal-bt-timegmt"/></td>
+          </tr>
+          <tr>
+              <td class="nbsp"><%= _('Level') %></td>
+              <td><div id="modal-bt-level"/></td>
+          </tr>
+          <tr>
+              <td class="nbsp"><%= _('Message') %></td>
+              <td><div class="pre-scollable-xy-modal" id="modal-bt-message"/></td>
```

Comment:
This is too small for any substential message, consider increasing the modal size:
![screenshot from 2016-01-27 23-58-21](https://cloud.githubusercontent.com/assets/1540531/12629919/a24b7976-c552-11e5-9ee4-b66087fe1bd8.png)